src/Modules/UI/Dnn.PersonaBar.UI/admin/personaBar/Bundle.Web/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "export-bundle",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "private": true,
   "scripts": {
     "build": "set NODE_ENV=production&&webpack -p",
-    "webpack": "webpack-dev-server -d --port 8070 --hot --inline --content-base dist/ --history-api-fallback",
+    "webpack": "set NODE_ENV=development&&webpack-dev-server -d --port 8070 --hot --inline --content-base dist/ --history-api-fallback",
```

Comment:
Interesting. I don't mind but is that necessary? Is the variable the used anywhere?

Reply:
it is used in webpack.config.js to switch source mapping on-off. If you run webpack-dev-server locally, you will have source map, in production you won't

Reply:
Correct, but the check is `const isProduction = process.env.NODE_ENV === "production";` I don't see the value `development` used

Reply:
You're right. It could be anything that's not "production", actually. I just set it to "development" to make it clear and coherent.

Reply:
If you could just remove it. We don't make this explicit anywhere else and it's redundant

Reply:
ok